Utilities/StaticAnalyzers/test/run_static_checks.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash -xe
+[ -z $WORKSPACE ] && (echo "WORKSPACE not set!"; exit 1)
+[ -z CMSSW_BASE ] && (echo "CMSSW_BASE not set!"; exit 1)
+: ${NCPU2:=1}
+
+export CMS_BOT_DIR=$WORKSPACE/cms-bot
+
+rm -rf $WORKSPACE/llvm-analysis-test; mkdir $WORKSPACE/llvm-analysis-test
+pushd $CMSSW_BASE/src/Utilities/StaticAnalyzers
```

Comment:
@iarspider , use  `Utilities/StaticAnalyzers/test`  so that we only build unit tests only